enterprise/backend/src/metabase_enterprise/api/routes.clj

```diff
@@ -58,6 +60,7 @@
    "/audit-app"                  (premium-handler metabase-enterprise.audit-app.api.routes/routes :audit-app)
    "/autodescribe"               (premium-handler 'metabase-enterprise.llm.api :llm-autodescription)
    "/billing"                    metabase-enterprise.billing.api.routes/routes
+   "/data-editing"               (premium-handler metabase-enterprise.data-editing.api/routes :table-data-editing)
```

Comment:
Route ends up like `/data-editing/table/...`. Can reorganize if we add other kinds of data editing.